starve.io-deobfuscate.user.js

```diff
@@ -61,7 +61,7 @@
                             if (window[s].toString().match(/new XMLHttpRequest/)) { // v15 1 match
                                 window.Client = s;
                                 deobmatch('Client', s);
-                            } else if (deobfunc.abbr === 'function\s?@(){SAVE;TUPT;this.text&&(@.globalAlpha=this.chat.o?1-this.chat.v:1,this.label||(this.label=create_message(scale,this.text)),@.drawImage(this.label,-this.label.width/2,-this.label.height/2-110*scale),this.chat.update()&&0==this.chat.o&&(this.text="",this.label=null));RESTORE}') {
+                            } else if (deobfunc.abbr === 'function @(){SAVE;TUPT;this.text&&(@.globalAlpha=this.chat.o?1-this.chat.v:1,this.label||(this.label=create_message(scale,this.text)),@.drawImage(this.label,-this.label.width/2,-this.label.height/2-110*scale),this.chat.update()&&0==this.chat.o&&(this.text="",this.label=null));RESTORE}') {
```

Comment:
For compatibility with Mozilla Firefox, there is no space between "function" and "@"